Vic2ToHoI4/Source/HOI4World/Map/ResourcesLink.h

```diff
@@ -0,0 +1,25 @@
+#ifndef RESOURCES_LINK_H
+#define RESOURCES_LINK_H
+
+
+
+#include "newParser.h"
+
+
+
+class ResourcesLink: commonItems::parser
+{
+	public:
+		explicit ResourcesLink(std::istream& theStream);
+
+		[[nodiscard]] auto getProvinceNum() const { return provinceNum; }
+		[[nodiscard]] auto getResources() const { return theResources; }
```

Comment:
const auto& for map. let the other side iterate through a pointer.

Reply:
Also did move mechanics here.